plugin/k8s_ingress/external.go

```diff
@@ -0,0 +1,115 @@
+/*
```

Comment:
Should this file be named `ingress.go`?  is `external.go` a remanent of k8s_external?